.github/workflows/pr-for-release.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: Pull Request on release/* Branch Push
+on:
+  push:
+    branches:
+      - "release/**"
+      - "!release/*"
```

Comment:
Could you explain this line?

Reply:
This line means that we want to create PR for any branch `release/networkservicemesh/*/v*.*.*`, but don't want to create PR for branch `release/v*.*.*`
`**` - include any character
`*` - include any character except `\`

Reply:
Got it.
This sounds like a delicate moment that I guess should be pointed by comment for this line in the file.

Reply:
ok